GETTING_STARTED_GUIDE.md

```diff
@@ -1,18 +1,15 @@
 ## Getting started guide
 
-### How to run assistant on your own machine
+### How to run assistant on your 2.12+ OpenSearch cluster
 Below are the set of steps to run OpenSearch and OpenSearch dashboards with the OpenSearch assistant and the query generation functionality in the Observability Log Explorer page correctly on the cluster.
-**Note** that the `feature/langchain` is the branch used in this guide.
 
-1. Follow steps here to setup docker for OpenSearch: https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/
-   1. Note: When running docker pull, use this command instead: `docker pull public.ecr.aws/w1m7p7g2/opensearch-reinvent2023:latest`
-   2. Note: This docker image may get incrementally updated over time.
-
-
-2. Follow steps here to setup docker to OpenSearch Dashboards: https://opensearch.org/docs/latest/install-and-configure/install-dashboards/docker/
-   1. Note: When running docker pull, use this command instead for OSD: `docker pull public.ecr.aws/w1m7p7g2/opensearch-dashboards-reinvent2023:latest`
-   2. Note: This docker image may get incrementally updated over time.
-   3. If you want to enable the chat assistant feature, set `assistant.chat.enabled` to `true` in the `opensearch_dashboards.yml` file.
+1. Setup a 2.12+ OpenSearch cluster with OpenSearch Dashboards by following the options here: https://opensearch.org/docs/latest/install-and-configure/
+   1. Note: If you are using a min distribution, there are required OpenSearch and OpenSearch Dashboards plugin to run the assistant.
+      1. Required OpenSearch plugins: ML-Commons, Flow Framework, and Observability
```

Comment:
i think https://github.com/opensearch-project/skills is also required? It contains PPL tools and other tools. Maybe also SQL plugin for executing PPL in query assist?